src/IssuesRepro/DIPS.Xamarin.Forms.IssuesRepro/DIPS.Xamarin.Forms.IssuesRepro/MainViewModel.cs

```diff
@@ -1,44 +1,43 @@
-﻿using System.Windows.Input;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Input;
 using Xamarin.Forms;
 
 namespace DIPS.Xamarin.Forms.IssuesRepro
 {
     public class MainViewModel
     {
-        public MainViewModel()
+        private static readonly Dictionary<string, Func<Page>> s_issues = new Dictionary<string, Func<Page>>
         {
-            NavigationCommand = new Command<string>(Navigate);
-        }
+            { "64", () => new Github64.Github64() },

```

Comment:
Alternatively, have a look at this: https://github.com/xamarin/Xamarin.Forms/blob/76353d15e654ef1bcf37fe55cd8c19e17a0e06b1/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/GitHub1355.cs#L17

This can also be used to search for issues :D